Clear/Interpreter.lean

```diff
@@ -103,7 +103,7 @@ mutual
 
       | .PrimCall prim args => evalPrimCall prim (reverse' (evalArgs fuel args.reverse s))
       | .Call f args        => evalCall fuel f (reverse' (evalArgs fuel args.reverse s))
-      | .Var id             => (s, s[id]!)
+      | .Var id             => (s, s[id]!!)
```

Comment:
Oh mylanata, do we still have the `notation s "[" x "]" "!"` stuff? The `!!` looks wrong ^^.

Reply:
We don't. It was taken over by the `GetElem` typeclass. Wasn't it you who suggested this workaround? :)

Reply:
https://github.com/NethermindEth/Clear/pull/5 does not address this. I am not sure what the best approach is here.